UG-Samples/Connectors/Segments/Bezier/BezierSegmentShape/Pages/_Layout.cshtml

```diff
@@ -0,0 +1,35 @@
+﻿@using Microsoft.AspNetCore.Components.Web
+@namespace BezierSegmentShape.Pages
+@addTagHelper *, Microsoft.AspNetCore.Mvc.TagHelpers
+
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="utf-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <base href="~/" />
+    <link rel="stylesheet" href="css/bootstrap/bootstrap.min.css" />
+    <link href="css/site.css" rel="stylesheet" />
+    <link href="BezierSegmentShape.styles.css" rel="stylesheet" />
+    <link href="_content/Syncfusion.Blazor.Themes/bootstrap5.css" rel="stylesheet" />
```

Comment:
<link href="_content/Syncfusion.Blazor.Themes/bootstrap5.css" rel="stylesheet" />

If you refer the separate Nuget use the below code.
<script src="_content/Syncfusion.Blazor.Core/scripts/syncfusion-blazor.min.js" type="text/javascript"></script>

Or use the whole Nuget
<script src="_content/Syncfusion.Blazor/scripts/syncfusion-blazor.min.js" type="text/javascript"></script>